pkg/reconciler/sequence/sequence.go

```diff
@@ -215,6 +216,16 @@ func (r *Reconciler) reconcileChannel(ctx context.Context, channelName string, c
 		}
 		logging.FromContext(ctx).Error(fmt.Sprintf("Failed to get Channel: %s/%s", p.Namespace, channelName), zap.Error(err))
 		return nil, err
+	} else if !equality.Semantic.DeepEqual(newChannel.Object["spec"], c.Object["spec"]) {
+		updatedChannel := c.DeepCopy()
```

Comment:
DeepCopy is not needed since Get() returns a new object.

Reply:
done, since channel in subscription is immutable, change to delete and recreate instead of update